packages/integration-test/test/lit-protocol.test.ts

```diff
@@ -0,0 +1,241 @@
+import { EthereumPrivateKeySignatureProvider } from '@requestnetwork/epk-signature';
+import { LitProtocolProvider } from '@requestnetwork/lit-protocol-cipher';
+import { RequestNetwork, Types, Utils } from '@requestnetwork/request-client.js';
+import { ethers } from 'ethers';
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+async function waitForConfirmation(request: any, maxAttempts = 10, delayMs = 1000): Promise<void> {
+  let attempts = 0;
+  while (attempts < maxAttempts) {
+    try {
+      const data = await request.getData();
+      if (
+        data.state === Types.RequestLogic.STATE.CREATED ||
+        data.state === Types.RequestLogic.STATE.PENDING
+      ) {
+        console.log(`Request confirmed with state: ${data.state}`);
+        return;
+      }
+      console.log(
+        `Attempt ${attempts + 1}: Request not confirmed yet. Current state: ${data.state}`,
+      );
+    } catch (error) {
+      console.log(`Attempt ${attempts + 1} failed:`, error);
+    }
+    await sleep(delayMs);
+    attempts++;
+  }
+  throw new Error(`Request not confirmed after ${maxAttempts} attempts`);
+}
+
+describe('Lit Protocol Integration Tests', () => {
+  let requestNetwork: RequestNetwork;
+  let litProvider: LitProtocolProvider;
+  let epkSignatureProvider: EthereumPrivateKeySignatureProvider;
+  let userWallet: ethers.Wallet;
+  let litClient: LitNodeClient;
+
+  const nodeConnectionConfig = {
+    baseURL: 'http://localhost:3000',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+
+  beforeAll(async () => {
+    // Create wallet
+    userWallet = new ethers.Wallet(
```

Comment:
Can't we just use `ethers.wallet.createRandom` to generate random wallets for test instead of using private key ?

Reply:
Random data for fuzz tests. Static data for unit and integration tests.

Reply:
Good point. I will revert it.

Reply:
Wait, I didn't change at all.